CMakeLists.txt

```diff
@@ -159,6 +160,7 @@ SET(${PROJECT_NAME}_ALGORITHM_HEADERS
   algorithm/compute-all-terms.hpp
   )
 
+IF(${BUILD_PYTHON_INTERFACE} STREQUAL "ON")
```

Comment:
Minor remark. For future, here you can directly  use IF(BUILD_PYTHON_INTERFACE).